### DISCUSSION TITLE
How can I remove the skip day

### DISCUSSION THREAD

**Discussion by louanfontenele** [EssentialsX/Essentials#4090] [Q&A] [answered]
Hello, I don't want to skip the day instantly, as I do to make it normal, I already use another plugin called Habor, I would love to make it normal like vanilla

I know that permission is: essentials.sleepingignored

Is there any way for me to remove this?

**Comment by triagonal** [comment, marked as answer]
If you're opped or have a wildcard permission like `*` or `essentials.*`, you'll need to negate (set to false) the permission you listed above in your permissions plugin. 

If you're using LuckPerms, then the command would be something like `/lp group owner permission set essentials.sleepingignored false`, given all opped/wildcard users are in the `owner` group.
